clang/include/clang/Sema/Sema.h

```diff
@@ -1698,6 +1698,9 @@ class Sema final {
     ChangedStateAtExit
   };
 
+  void DiagnoseMissingFormatAttributes(const FunctionDecl *FDecl,
```

Comment:
This declaration is placed among functions implemented in `SemaAttr.cpp`, as you might see around line 1296, but its implementation is placed in `SemaDeclAttr.cpp`. Declarations of functions in this file have their own section in `Sema.h`, which starts around line 3690. This discrepancy should be fixed one way or another.